ironmon_tracker/constants/MoveData.lua

```diff
@@ -1678,7 +1678,7 @@ MoveData.MOVES_MASTER_LIST = {
 		pp = "5",
 		accuracy = "100",
 		category = MoveData.MOVE_CATEGORIES.PHYSICAL,
-		description = "User faints, even if the attack fails or misses. Inflicts regular damage."
+		description = {"User faints, even if the attack fails or misses. Inflicts regular damage. Halves the target's defense for damage calculation.", "User faints, even if the attack fails or misses. Inflicts regular damage. Halves the target's defense for damage calculation.", "User faints, even if the attack fails or misses. Inflicts regular damage. Halves the target's defense for damage calculation.", "User faints, even if the attack fails or misses. Inflicts regular damage. Halves the target's defense for damage calculation.", "User faints, even if the attack fails or misses. Inflicts regular damage."}
```

Comment:
Defense should be capitalized here (as stats are generally capitalized in move descriptions)